src/Library/demos/Menu Button/code.rs

```diff
@@ -0,0 +1,23 @@
+use crate::workbench;
+use gtk::prelude::*;
+use gtk::{Button, Switch};
+
+pub fn main() {
+    let circular_switch: Switch = workbench::builder()
+        .object("circular_switch")
+        .expect("Failed to get circular_switch");
+    let primary_button: Button = workbench::builder()
+        .object("primary")
+        .expect("Failed to get primary_button");
+    let secondary_button: Button = workbench::builder()
```

Comment:
That should be a `MenuButton` not a `Button`

Reply:
Check now pls

Reply:
Have you tried the code?
it doesn't look like it would work. I also don't think it's formatted.

Reply:
I forgot to add an import, it works on my side now + did the formatting.

Pls check :)

Reply:
Doesn't look formatted to me :)

Please run the code in Workbench, that will format it.
Then copy the result to `code.rs`.

See here:
https://github.com/sonnyp/Workbench/pull/554#discussion_r1309819779

Reply:
Also, there are code conflicts now, let me know if you need help with solving them.

Reply:
The code conflict occured cuz i added my name in About.js, should i remove that?

Reply:
Also, I ran the code and copy-pasted it as you asked. Let me know if its correctly formatted now

Reply:
The code looks good now.

you don’t have to remove your name, you should resolve the conflict in a way that all of the additions are there. You can give it a try by clicking on “Resolve conflicts” on this page

Reply:
I've done it for you.
You can see how in this screencast, so you can try next time on your own :)
[screencast](https://github.com/sonnyp/Workbench/assets/30049909/534cda8b-8b66-411c-8d58-90fee73694f7)